pages/governance.html

```diff
@@ -10,193 +10,106 @@
 <div class="container-text">
 	<h3>Legal status</h3>
 	<p>
-		On its own, Godot has no legal status. Godot exists as a member project of
-		the <a href="https://sfconservancy.org">Software Freedom Conservancy</a>
-		(SFC). The SFC provides a home and infrastructure for Free, Libre, and Open
-		Source Software (FLOSS) projects by aggregating the work necessary for
-		running a non-profit organization including accounting and legal compliance.
-		For more information about what the SFC provides, please refer to <a
-			href="https://sfconservancy.org/projects/services/">their website</a>.
+		The Godot Engine is financially supported by the <a href="https://godot.foundation">Godot Foundation</a>, a non-profit organization formed on August 23rd, 2022 via the KVK (number 87351919). The Godot Foundation is responsible for managing donations made to the Godot Project and ensuring that such donations are used to enhance the Godot Engine. The Godot Foundation is a legally independent organization and does not own the Godot Project. In the past, the Godot Project existed as a member project of the <a href="https://sfconservancy.org"><strong>Software Freedom Conservancy</strong></a>.
```

Comment:
This paragraph first refers to "The Godot Engine", but then "The Godot Project". The term "Godot Project" is never introduced. Might be worth addressing that. Also later in the document "The Godot project" is used without the same capitalization.

Reply:
It should always be "Godot Project" IMO (and properly introduce the term first). The "Godot Engine" term is too narrow, there are other areas of the ecosystem also supported by the foundation that goes beyond the engine (software) itself. Like the communities, asset library, events, etc.